pages/index.tsx

```diff
@@ -54,18 +61,32 @@ export default function Home() {
         </div>
       </main>
 
+      <div>
+        <h1 className="text-center text-3xl font-bold">{NEW_USER}</h1>
+        <div className="flex flex-wrap w-full mx-auto justify-center">
+          {Array.from({ length: NUM_MEMBERS_NUMBER }).map((_, i) => (
```

Comment:
just add a comment here regarding use of  `Array.from`

Reply:
added